generate_data.py

```diff
@@ -355,6 +367,10 @@ def _idx_2_domain_label(index):
     parser.add_argument("-a", "--alpha", type=float, default=0)
     parser.add_argument("-ls", "--least_samples", type=int, default=40)
 
+    # Flower partitioner
+    parser.add_argument("--flower_partitioner_class", type=str, default="")
+    parser.add_argument("--flower_partitioner_kwargs", type=str, default="{}")
```

Comment:
Maybe some discriptions here and `data/README.md`? Also, the arugment name length is too long, need abbreviation for them.